keylime/tenant.py

```diff
@@ -598,59 +598,75 @@ def do_cvadd(self) -> None:
         )
 
         if response.status_code == 503:
+            # EVALUATE DELETION
```

Comment:
I agree that with the exception the log can now be dropped

Reply:
Well, I left all these `EVALUATE DELETION` because most of these will require changes to CI testing (unfortunately, our current testing mechanism is reliant on **a lot** of `grep` into logs, and I intend to fix it (Probably in a PR "there" - https://github.com/RedHat-SP-Security/keylime-tests - parallel to this one)